content/tutorials/how-to-deposit-erc20/10.index.md

```diff
@@ -75,7 +75,7 @@ dotenv.config();
 
 // HTTP RPC endpoints
 const L1_RPC_ENDPOINT = process.env.L1_RPC_ENDPOINT || ""; // or an RPC endpoint from Infura/Chainstack/QuickNode/etc.
-const L2_RPC_ENDPOINT = process.env.L2_RPC_ENDPOINT || "https://testnet.era.zksync.dev"; // or the zkSync Era mainnet
```

Comment:
You can use the zk_parser tag - `%%zk_testnet_rpc_url%%` for the rpc url so that it is automatically injected. This will make making changes in the future easier.